src/modules/deposit/dto.ts

```diff
@@ -16,13 +17,19 @@ export class GetDepositsQuery {
   status: "filled" | "pending";
 
   @IsOptional()
-  @IsNumberString({ no_symbols: true })
-  @ApiProperty({ example: "10", required: false })
+  @IsInt()
+  @Min(1)
+  @Max(100)
```

Comment:
Restrict `?limit` to max 100. The FE client doesn't request more than 50 deposits per table page